zellij-server/src/pty.rs

```diff
@@ -114,6 +117,12 @@ pub(crate) fn pty_thread_main(mut pty: Pty, layout: Box<LayoutFromYaml>) {
             PtyInstruction::UpdateActivePane(pane_id, client_id) => {
                 pty.set_active_pane(pane_id, client_id);
             }
+            PtyInstruction::FocusOnTab(tab_index, client_id) => {
+                pty.bus
+                    .senders
+                    .send_to_screen(ScreenInstruction::GoToTab(tab_index, Some(client_id)))
```

Comment:
Have both Instructions named similarly? They seem to be doing the same thing? So maybe
`PtyInstruction::FocusOnTab` and `ScreenInstruction::FocusOnTab` ?

Reply:
Thanks for the comment!
Yes, it does the same thing. It's just simple wrapper instruction.

> Maybe FocusTab would be sufficient here? I am not sure.

Similar to the feedback above, I thought it would be good to rename the Instruction.
Instead of `FocusOnTab`, how about unifying with `GoToTab`? (like `ScreenInstruction::GoToTab`)

Reply:
I like it. That is a good idea!